openedx/features/journals/apps.py

```diff
@@ -0,0 +1,11 @@
+"""Application Configuration for Journals."""
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class JournalsConfig(AppConfig):
```

Comment:
As @nasthagiri mentioned.  We should upgrade this AppConfig file to leverage the plugins architecture :)

Reply:
fixed